AUTHORS.md

```diff
@@ -0,0 +1,30 @@
+# Authors
+
+<!-- List format (alphabetical order): Surname, Name. Employer/Affiliation -->
+
+* Benedicic, Lucas. ETH Zurich - CSCS
```

Comment:
Did you double-check that all people from both branches are there?

Reply:
Yes, the authors list is up to date.